sample/src/main/java/io/ona/kujaku/sample/activities/CaseRelationshipActivity.java

```diff
@@ -145,6 +150,24 @@ private void toggleDrawingArrowsShowingRelationship() {
         } else {
             kujakuMapView.addLayer(arrowLineLayer);
             drawArrowsBtn.setText(R.string.disable_arrows_showing_relationship);
+
+            new Thread(() -> {
+                try {
+                    Thread.sleep(5000);
```

Comment:
Why not display a button at the bottom for changing to another focus area. So that user is aware what is happening.

If you decide with approach then animate to the new focus area .

Reply:
Cool, that's a good idea